plugins/bap/utils/ida.py

```diff
@@ -86,11 +86,12 @@ def output_types(out):
 @service.provider('brancher')
 def output_branches(out):
     """Dump static successors for each instruction """
+    out.write('(')
```

Comment:
The parentheses are added to combine all of the info into one sexp list, which makes the parsing in bap cleaner.